pnpm-lock.yaml

```diff
@@ -1949,6 +1949,9 @@ importers:
       '@remotion/studio-shared':
         specifier: workspace:*
         version: link:../studio-shared
+      formidable:
+        specifier: ^3.5.1
+        version: 3.5.1
```

Comment:
Can we make this work without dependencies?  
Unfortunately this adds 6 dependencies to a lean package otherwise.
I also mentioned this in the specification for this issue